lighthouse/Cargo.toml

```diff
@@ -7,7 +7,7 @@ autotests = false
 rust-version = "1.69.0"
 
 [features]
-default = ["slasher-lmdb"]
+default = ["slasher-redb"]
```

Comment:
Before merging we should change the default back to LMDB (for backwards compat).